CIP-PTR-ADDR/CPS-????.md

```diff
@@ -0,0 +1,41 @@
+---
+CPS: ?
```

Comment:
```suggestion
CPS: 2 
```

Reply:
grrr, I thought we had changed this already 😖